<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleWindow</class>
 <widget class="QWidget" name="SimpleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>1281</width>
     <height>641</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>82</y>
     <width>1101</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: #151c26;
	border-radius: 20px;
	border: 1px solid #44eb9b;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="previousButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #44eb9b;
	color: #151c26;
	border-radius: 10px;
	border-image: url(:/src/image/previous.png) stretch; 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: #44eb9b;
	background-color: rgb(74, 71, 169);
	border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="output">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>135</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: #44eb9b;
	font-size: 17px;
  	alignment: 'AlignVCenter | AlignRight';
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seven">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>212</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border: 1px solid rgb(74, 71, 169);
	font-size: 17px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	border: 1px solid rgb(68, 235, 155);
	color: rgb(68, 235, 155);
}</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eight">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>212</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border: 1px solid rgb(74, 71, 169);
	font-size: 17px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	border: 1px solid rgb(68, 235, 155);
	color: rgb(68, 235, 155);
}</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nine">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>212</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border: 1px solid rgb(74, 71, 169);
	font-size: 17px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	border: 1px solid rgb(68, 235, 155);
	color: rgb(68, 235, 155);
}</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="six">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>291</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border: 1px solid rgb(74, 71, 169);
	font-size: 17px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	border: 1px solid rgb(68, 235, 155);
	color: rgb(68, 235, 155);
}</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="four">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>291</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border: 1px solid rgb(74, 71, 169);
	font-size: 17px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	border: 1px solid rgb(68, 235, 155);
	color: rgb(68, 235, 155);
}</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="five">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>291</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border: 1px solid rgb(74, 71, 169);
	font-size: 17px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	border: 1px solid rgb(68, 235, 155);
	color: rgb(68, 235, 155);
}</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="three">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>370</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border: 1px solid rgb(74, 71, 169);
	font-size: 17px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	border: 1px solid rgb(68, 235, 155);
	color: rgb(68, 235, 155);
}</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="one">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>370</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border: 1px solid rgb(74, 71, 169);
	font-size: 17px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	border: 1px solid rgb(68, 235, 155);
	color: rgb(68, 235, 155);
}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="two">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>370</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border: 1px solid rgb(74, 71, 169);
	font-size: 17px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	border: 1px solid rgb(68, 235, 155);
	color: rgb(68, 235, 155);
}</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="geomOp">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>450</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
   </property>
   <property name="text">
    <string>Geom Op</string>
   </property>
  </widget>
  <widget class="QPushButton" name="baseOp">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>450</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
   </property>
   <property name="text">
    <string>Base Op</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zero">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>450</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border: 1px solid rgb(74, 71, 169);
	font-size: 17px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	border: 1px solid rgb(68, 235, 155);
	color: rgb(68, 235, 155);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QFrame" name="groupBase">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>187</y>
     <width>291</width>
     <height>341</height>
    </rect>
   </property>
   <widget class="QPushButton" name="mod">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>114</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>198</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divide">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>114</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bracketOpen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiply">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>198</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pow">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>114</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bracketClose">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ownXButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
    </property>
    <property name="text">
     <string>Own X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushPoint">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 15px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="groupGeom">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>202</y>
     <width>351</width>
     <height>281</height>
    </rect>
   </property>
   <widget class="QPushButton" name="cos">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tan">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>96</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="acos">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>96</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="atan">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>12</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sqrt">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>96</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>12</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="asin">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ln">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>12</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushEqual">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>627</x>
     <y>540</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AC">
   <property name="geometry">
    <rect>
     <x>507</x>
     <y>540</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
  </widget>
  <widget class="QLabel" name="ownX1">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>130</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: #44eb9b;
	background-color: rgb(74, 71, 169);
	border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ownX2">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>140</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: #44eb9b;
	font-size: 17px;
  	alignment: 'AlignVCenter | AlignRight';
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="plusX">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>120</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-weight: bold;
	color: rgb(74, 71, 169);
	background-color: rgb(68, 235, 155);
	border-style: solid;
	border-width: 5px;
	border-radius: 37px;
}

QPushButton:pressed {
	background-color: rgb(74, 71, 169);
	color: rgb(68, 235, 155);
}</string>
   </property>
   <property name="text">
    <string>Include X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushGraph">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>96</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Graph</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
